rosidl_typesupport_microxrcedds_c/resource/msg__type_support_c.c.em

```diff
@@ -317,6 +351,17 @@ size_t get_serialized_size_@('__'.join([package_name] + list(interface_path.pare
     size_t item_size = sizeof(ros_message->@(member.name).data[0]);
     current_alignment += ucdr_alignment(current_alignment, MICROXRCEDDS_PADDING) + MICROXRCEDDS_PADDING;
     current_alignment += ucdr_alignment(current_alignment, item_size) + (sequence_size * item_size);
+@[      elif isinstance(member.type.value_type, NamespacedType)]@
+    const size_t sequence_size = ros_message->@(member.name).size;
+
+    current_alignment += ucdr_alignment(current_alignment, MICROXRCEDDS_PADDING) + MICROXRCEDDS_PADDING;
+    
+    for(size_t i = 0; i < sequence_size; i++){
+      size_t element_size = ((const message_type_support_callbacks_t *)(
+        ROSIDL_TYPESUPPORT_INTERFACE__MESSAGE_SYMBOL_NAME(rosidl_typesupport_microxrcedds_c, @(', '.join(member.type.value_type.namespaced_name()))
+        )()->data))->get_serialized_size(&ros_message->@(member.name).data[i]);
+      current_alignment += ucdr_alignment(current_alignment, element_size) + element_size;
```

Comment:
At that point, we are not able to compute properly the alignment because it depends on the `NamespacedType`'s kind, but we can compute a `max_type_size` with the following approximation. 
```suggestion
      uint8_t alignment_size = (element_size < MICROXRCEDDS_PADDING) ? element_size : MICROXRCEDDS_PADDING;
      current_alignment += ucdr_alignment(current_alignment, alignment_size) + element_size;
```

Reply:
Done